### PR TITLE
Allow additional properties

### LINKED ISSUE #316: additionalProperties to true
We are currently trying to implement Prefect but need to set an additional property to get it properly working. I tried to create a PR for this but do not have the proper permissions.

Could you change `"additionalProperties": false` to `"additionalProperties": true`?

Thanks in advance!

### PR DESCRIPTION
**Pull request by fabrice-toussaint**
Closes #316

### CODE REVIEW

**Comment by jamiezieziula** [comment]
Hi @fabrice-toussaint, can you give us some more insight on the additional property that you would like to add and what it would be mapped to in your use case?

**Comment by fabrice-toussaint** [comment]
@jamiezieziula we are trying to implement Prefect using vcluster to spin up a temporary environment. We want to be able to enable / disable specific charts (depending on the temporary environment). The other services that we use (i.e. postgres) allow us to set an additional property (i.e. enabled) so that we can include / exclude them.

File: charts/prefect-server/values.schema.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema#",
   "type": "object",
-  "additionalProperties": false,
+  "additionalProperties": true,
```
Comment (jimid27) [inline charts/prefect-server/values.schema.json:4]: For context this means that something like:
Screenshot 2024-05-01 at 2 48 44 PM

is great.  But something like:
Screenshot 2024-05-01 at 2 49 21 PM

is still not permitted which makes sense for these use cases.

**Review by jimid27 — APPROVED** [review]
lgtm - appreciate you explaining your use case!

File: charts/prefect-server/values.schema.json (hunk 1 above)
Comment (fabrice-toussaint) [reply charts/prefect-server/values.schema.json:4]: Yes exactly that would work perfectly for us

**Pull request merged**